documentation/docs/guide/content/using-custom-post-types.md

```diff
@@ -0,0 +1,206 @@
+# Using Custom Post Types
+
+Welcome to this tutorial on using custom post types with the Block Theme. The goal of this tutorial is to show you how to setup and use a custom post type from your WordPress dashboard. But before we get into that, let's talk about custom post types and why they're useful.
+
+## What are custom post types?
+
+### Introduction
+
+A custom post type is used when the regular WordPress posting system is not suitable for a task. The regular WordPress posting system allows you to create news/blog posts and organize/manage them from the WordPress Dashboard. On the public side of your website, these posts can then be displayed to the user as a feed. This functionality is ideal for simple scenarios where you wish to keep users up to date. However, maybe you require an events calendar that sorts by event date, or a product catalogue. Perhaps you would like to use the regular posting system for news/blog updates but need a different post type for a different purpose. This is when custom post types become useful.
+
+### Custom Post Type UI
+
+The Custom Post Type UI plugin is one of the required plugins for the Block Theme. This is a hint that it's very important. It allows us to create and modify custom post types, and it gives us a long list of options for them. It allows us to:
+
+* Create, edit, and delete custom post types
+* Control which features are available for the post type
+* Change how the post type title appears in fields and menus
+* Control several advanced settings that are beyond the scope of this tutorial
+
+In addition to creating and managing custom post types, you can also create and manage taxonomies with the Custom Post Type UI. We will discuss this in a separate tutorial - let's stick to custom post types for now.
+
+## Making and using a custom post type
+
+### 1. The Scenario
+
+For the purposes of this tutorial, we will pretend that we are creating an environmental protection website. On this website, we need a regular news feed to inform the public of our progress and a separate alert feed with important updates about environmental disasters and missing animals. The alert feed will show up higher on the homepage, with bold colours that inform the page visitor of its importance.
+
+Note: The block theme includes a banner feature that can be used to show a single alert as a banner at the top of your homepage. However, if you wish to show multiple alerts at once, this method would be a good solution.
```

Comment:
Suggest changing "this method" to something more describing such as "custom post type method".
As "this" in the sentence context can be confused with the banner feature.